hc/settings.py

```diff
@@ -133,7 +133,7 @@
 
 USE_TZ = True
 
-SITE_ROOT = "https://hc-kalii.herokuapp.com"
+SITE_ROOT = "http://localhost:8000"
```

Comment:
This config is required for Heroku, please change it back

Reply:
Change made pushing now